src/components/Movies.js

```diff
@@ -3,7 +3,7 @@ import MovieCard from './MovieCard'
 import '../styles/Movies.scss'
 
 const Movies = props => {
-  const sortedMovies = props.movies.movies.sort((a, b) => (a.release_date < b.release_date) - (a.release_date > b.release_date))
+  const sortedMovies = props.movies.sort((a, b) => (a.release_date < b.release_date) - (a.release_date > b.release_date))
```

Comment:
Nice adjustment to allow our fetch data to update the DOM

Reply:
Love this sorting!